### PR TITLE
Fix formatting issue in crd-upgrade-123.md

### PR DESCRIPTION
**Pull request by Andygol**
## Description

This PR fixes formatting errors in boilerplate

## Reviewers

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Comment by kfaseela** [comment]
Could you please link to a page that renders this with and without the fix?

**Comment by Andygol** [comment]
> Could you please link to a page that renders this with and without the fix?

@kfaseela, you may find incorrect text rendering here 👉 https://preliminary.istio.io/latest/docs/ambient/upgrade/helm/#base-components

**Comment by kfaseela** [comment]
> > Could you please link to a page that renders this with and without the fix?
> 
> @kfaseela, you may find incorrect text rendering here 👉 https://preliminary.istio.io/latest/docs/ambient/upgrade/helm/#base-components

With your changes : https://deploy-preview-15873--preliminary-istio.netlify.app/latest/docs/ambient/upgrade/helm/#base-components

**Comment by kfaseela** [comment]
@Andygol you will have to fix the gen check failure

**Comment by kfaseela** [comment]
Please take a look at the lint failures at : https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/15873/lint_istio.io/1854447690931245056

**Comment by Andygol** [comment]
@kfaseela I've figure out what is the root cause of the issue and preparing a fix ⛑️ 

IMO gen procedure incorrectly process multiline shell commands such as for loop 🤷‍♂️ 


Adding extra `$` on each line is the way to overcome the issue but it would be nice to amend gen scrips

🙅 This is incorrect syntax for a multiline command, but it passes lint/gen routines 

```sh
$ for crd in $(kubectl get crds -l chart=istio -o name && kubectl get crds -l app.kubernetes.io/part-of=istio -o name)
$ do
$    kubectl label "$crd" "app.kubernetes.io/managed-by=Helm"
$    kubectl annotate "$crd" "meta.helm.sh/release-name=istio-base" # replace with actual Helm release name, if different from the documentation default
$    kubectl annotate "$crd" "meta.helm.sh/release-namespace=istio-system" # replace with actual istio namespace
$ done
``` 

🤌 This syntax is fine, but it does not pass lint

```sh
$ for crd in $(kubectl get crds -l chart=istio -o name && kubectl get crds -l app.kubernetes.io/part-of=istio -o name)
  do
     kubectl label "$crd" "app.kubernetes.io/managed-by=Helm"
     kubectl annotate "$crd" "meta.helm.sh/release-name=istio-base" # replace with actual Helm release name, if different from the documentation default
     kubectl annotate "$crd" "meta.helm.sh/release-namespace=istio-system" # replace with actual istio namespace
  done
```

File: content/en/boilerplates/crd-upgrade-123.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ You can resolve this with a one-time migration using the following `kubectl` com
 
 {{< text syntax=bash snip_id=adopt_legacy_crds >}}
 $ for crd in $(kubectl get crds -l chart=istio -o name && kubectl get crds -l app.kubernetes.io/part-of=istio -o name)
-  do
-     kubectl label "$crd" "app.kubernetes.io/managed-by=Helm"
-     kubectl annotate "$crd" "meta.helm.sh/release-name=istio-base" # replace with actual Helm release name, if different from the documentation default
-     kubectl annotate "$crd" "meta.helm.sh/release-namespace=istio-system" # replace with actual istio namespace
-  done
+$ do
```
[the remark this answers is not in the document]
Comment (Andygol) [reply content/en/boilerplates/crd-upgrade-123.md:12]: gen script process line with `$` and creates shell code that you may `copy` from the code block, lines without `$` won't be added to the copied code 🤷‍♂️

**Comment by Andygol** [comment]
@kfaseela checks have been passed 🎉 

Feel free to /lgtm /approve

**Comment by kfaseela** [comment]
I think this is fine as from your deploy preview, if I use the copy button it does not copy these $ symbols, and I get the right script to be exceuted: https://deploy-preview-15873--preliminary-istio.netlify.app/latest/docs/ambient/upgrade/helm/

**Review by kfaseela — APPROVED** [review]

**Comment by Andygol** [comment]
`$` at the start of each lines confuses me a little bit 🤷‍♂️ 

image

**Comment by kfaseela** [comment]
> `$` at the start of each lines confuses me a little bit 🤷‍♂️
> 
> image

you are welcome to fix the gen script :)

**Pull request merged**